packages/nvidia_nat_core/tests/nat/front_ends/auth_flow_handlers/test_console_flow_handler.py

```
@@ -196,3 +200,98 @@ def construct_oauth_client(self, cfg):
     # Verify the error message contains the original exception information
     error_message = str(exc_info.value)
     assert "Invalid OAuth client configuration" in error_message
+
+
+# ---------------------------------------------------------------------------
+# ConsoleFrontEndPlugin – preflight authentication
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)